### PR TITLE
Add price graph

### PR DESCRIPTION
**Pull request by DeDiamondPro**
Very cool graph that you opens by pressing p while hovering over an item.

### CODE REVIEW

**Review by nopothegamer — APPROVED** [review]
looks good

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiPriceGraph.java [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+    private long firstTime;
+    private long lastTime;
+    private Float lowestValue = null;
+    private String itemName;
+    private final String itemId;
+    private int guiLeft;
+    private int guiTop;
+    private ItemStack itemStack = null;
+    private boolean loaded = false;
+    /**
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiPriceGraph.java:50]: wtf javadocs in neu - illegal

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 2]
```diff
@@ -50,6 +50,7 @@
 import java.util.regex.Pattern;
 
 public class Utils {
+
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:53]: no

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 3]
```diff
@@ -472,18 +473,21 @@ public static String prettyCase(String str) {
...
-        put("MYTHIC", rarityArrC[5]);
-        put("SPECIAL", rarityArrC[6]);
-        put("VERY SPECIAL", rarityArrC[7]);
-        put("DIVINE", rarityArrC[8]);
-        // put("DIVINE", rarityArrC[9]);
-    }};
+    public static final HashMap<String, String> rarityArrMap = new HashMap<>();
+
+    static {
+        rarityArrMap.put("COMMON", rarityArrC[0]);
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:479]: no

**Review by IRONM00N — CHANGES_REQUESTED** [review]

**Review by nopothegamer — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]
lgtm

**Pull request merged**